src/include/platform/internal/GenericPlatformManagerImpl_POSIX.cpp

```diff
@@ -56,8 +56,6 @@ System::LayerSocketsLoop & SystemLayerSocketsLoop()
 template <class ImplClass>
 CHIP_ERROR GenericPlatformManagerImpl_POSIX<ImplClass>::_InitChipStack()
 {
-    mChipStackLock = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
Why the change to move this out of _InitChipStack?  Now _InitChipStack and _Shutdown are asymmetric and internalization of the stack is impossiblle.

Reply:
I meant to revert this, thanks for catching.